.vscode/launch.json

```diff
@@ -12,6 +12,22 @@
 			"program": "${workspaceFolder}/server/main",
 			"cwd": "${workspaceFolder}"
         },
+        {
```

Comment:
The indentation of this file looks broken.

Reply:
Thanks, pushed a fix, and also cleaned up tab consistency in a few files.